src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -51,6 +51,14 @@
           <span class='table-warning'>This node has incoherent dependencies</span>
         </td>
       </tr>
+      <tr>
+        <td>
+        <span class="align-self-center" style="padding-right: 1rem;"><fa-icon icon="redo"></fa-icon></span>
```

Comment:
NIT: indent these spans

Reply:
Copy paste. I'll fix the other instances of this too.